File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U40EmrHistoryMedicalRecordInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class OCS2015U40EmrHistoryMedicalRecordInfo
    {
        private String _sysId;
        private String _created;
        private String _memo;
        private String _activeFlg;
        private String _version;
        private String _author;
        private String _emrRecordId;

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String Created
        {
            get { return this._created; }
            set { this._created = value; }
        }

        public String Memo
        {
            get { return this._memo; }
            set { this._memo = value; }
        }

        public String ActiveFlg
        {
            get { return this._activeFlg; }
            set { this._activeFlg = value; }
        }

        public String Version
        {
            get { return this._version; }
            set { this._version = value; }
        }

        public String Author
        {
            get { return this._author; }
            set { this._author = value; }
        }

        public String EmrRecordId
        {
            get { return this._emrRecordId; }
            set { this._emrRecordId = value; }
        }

        public OCS2015U40EmrHistoryMedicalRecordInfo() { }

        public OCS2015U40EmrHistoryMedicalRecordInfo(String sysId, String created, String memo, String activeFlg, String version, String author, String emrRecordId)
        {
            this._sysId = sysId;
            this._created = created;
            this._memo = memo;
            this._activeFlg = activeFlg;
            this._version = version;
            this._author = author;
            this._emrRecordId = emrRecordId;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U40EmrHistoryMedicalRecordInfo.java
package nta.med.data.model.ihis.emr;

import java.math.BigDecimal;
import java.util.Date;

public class OCS2015U40EmrHistoryMedicalRecordInfo {
	private Integer recordId;
	private String sysId;
	private Date created;
	private String memo;
	private BigDecimal activeFlg;
	private Integer version;
	private String author;

	public OCS2015U40EmrHistoryMedicalRecordInfo(Integer recordId,
			String sysId, Date created, String memo, BigDecimal activeFlg,
			Integer version, String author) {
		super();
		this.recordId = recordId;
		this.sysId = sysId;
		this.created = created;
		this.memo = memo;
		this.activeFlg = activeFlg;
		this.version = version;
		this.author = author;
	}

	public Integer getRecordId() {
		return recordId;
	}

	public void setRecordId(Integer recordId) {
		this.recordId = recordId;
	}

	public String getSysId() {
		return sysId;
	}

	public void setSysId(String sysId) {
		this.sysId = sysId;
	}

	public Date getCreated() {
		return created;
	}

	public void setCreated(Date created) {
		this.created = created;
	}

	public String getMemo() {
		return memo;
	}

	public void setMemo(String memo) {
		this.memo = memo;
	}

	public BigDecimal getActiveFlg() {
		return activeFlg;
	}

	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}

	public Integer getVersion() {
		return version;
	}

	public void setVersion(Integer version) {
		this.version = version;
	}

	public String getAuthor() {
		return author;
	}

	public void setAuthor(String author) {
		this.author = author;
	}

}